src/wallet/wallet.h

```diff
@@ -533,6 +533,11 @@ class CAccountingEntry
     std::vector<char> _ssExtra;
 };
 
+enum CreateTransactionFlags {
+    CREATE_TX_DEFAULT     = 0,
```

Comment:
I think the `CREATE_TX_DEFAULT` is okay. Passing just a `0`(int) would require casting. "not bip125" is also not true because in future you could combine flags (Don't SIGN & RBF)?